plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisJedisManager.java

```diff
@@ -81,6 +81,7 @@ private JedisPool createConsumer(HostAddress host)
                 host.getHostText(),
                 host.getPort(),
                 toIntExact(redisConnectorConfig.getRedisConnectTimeout().toMillis()),
+                redisConnectorConfig.getRedisUser(),
```

Comment:
Do we have any integration tests to ensure that - the userName is being used ?

Reply:
No, we haven't. I suppose we need newer Redis version (>= 6) to ensure that. Could you please review #12281 first?